test/acceptance/service-broker-api.instances.docker.spec.js

```diff
@@ -205,6 +205,29 @@ describe('service-broker-api', function () {
             });
         });
 
+        it('returns 410 Gone', function () {
+          mocks.docker.inspectContainer(instance_id, {
```

Comment:
not that it matters, but shouldnt the second param (options) to this method be `{}` or `undefined` ? To help the readability.